### PR TITLE
Feature/uuids as primary keys

### PR DESCRIPTION
**Pull request by jkp85**

### CODE REVIEW

**Comment by jkp85** [comment]
This PR should be thoroughly tested before merge. All details how to test are in README file.

File: servers/tests/test_views.py [hunk 1]
```diff
@@ -400,17 +399,17 @@ def setUp(self):
...
     def test_list(self):
         model = ModelFactory(server__project=self.project)
         stats = ServerStatisticsFactory(server=model.server)
         url = reverse('serverstatistics-list', kwargs={
             'namespace': self.project.get_owner_name(),
-            'project_pk': self.project.hashid,
+            'project_pk': str(self.project.pk),
             'server_type': 'models',
-            'server_pk': model.hashid
+            'server_pk': str(model.pk)
```
Comment (AbhishikthaReddy) [inline servers/tests/test_views.py]: Hi Rafal! why did we encode the primary key of project, env, workspace, model and jobs previously, why are we not doing that

Comment (jkp85) [inline servers/tests/test_views.py]: Hi. Previously we've used [http://hashids.org/python/](hashids) for hashing integer database ids, now we are using uuids and this is no longer necessary.

Comment (AbhishikthaReddy) [inline servers/tests/test_views.py]: gotcha! thanks!

**Pull request closed without merging**